src/Presentation/RestApi/Resources/config/services.yaml

```diff
@@ -1,6 +1,14 @@
 parameters: ~
 
 services:
+    _instanceof:
+        SprykerSdk\Sdk\Presentation\RestApi\Controller\v1\BaseController:
+            calls:
```

Comment:
Why does it go not via constructor?

Reply:
It will be easier to register new controllers, because I won't need to specify this dependency